Lares/Assets/Player/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,202 @@
+using System.Collections;
+using Unity.VisualScripting;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace Lares
+{
+
+    public class PlayerController : MonoBehaviour
+    {
+        private PlayerInput _input;
+        private Vector3 _moveDirection;
+        private Rigidbody _rigidbody;
+        private Coroutine _moveCoroutine;
+
+        private Vector3 _inclineHeight;
+
+        private bool _isGrounded
+        {
+            set
+            {
+                _isGroundedInternal = value;
+                if (!_isGroundedInternal)
+                    _movementForce *= _inAirMovementModifier;
+                else
+                    _movementForce /= _inAirMovementModifier;
+            }
+
+            get { return _isGroundedInternal; }
+        }
+
+        private bool _isGroundedInternal;
```

Comment:
Can make IsGrounded public - it may be useful for external things, maybe stopping the player from doing an interaction (wouldn't be on PlayerController, in my opinion) if not on the ground?

To protect the value, you can make the setter private:
```suggestion
        public bool IsGrounded
        {
            private set
            {
                _isGrounded = value;
                if (!_isGrounded)
                    _movementForce *= _inAirMovementModifier;
                else
                    _movementForce /= _inAirMovementModifier;
            }

            get { return _isGrounded; }
        }

        private bool _isGrounded;
```